Tests/OptimizelyTests-MultiClients/LoggerTests_MultiClients.swift

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright 2021, Optimizely, Inc. and contributors 
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");  
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at   
+// 
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import XCTest
+
+class LoggerTests_MultiClients: XCTestCase {
+
+    override func setUpWithError() throws {
```

Comment:
Can we add `base` class and put these two `overrided` methods in there and extend classes from that `base` class. I see these two methods at many locations.

Reply:
Most test cases will use own tasks in these methods. I'll clean up these empty methods.